Resources/Prototypes/_NF/Entities/Objects/Weapons/Guns/Ammunition/Projectiles/shotgun.yml

```diff
@@ -0,0 +1,23 @@
+- type: entity
+  id: Pellet410Shotgun
+  name: pellet (.410)
+  noSpawn: true
+  parent: BaseBullet
+  components:
+  - type: Sprite
+    sprite: Objects/Weapons/Guns/Projectiles/projectiles2.rsi
+    state: buckshot
+  - type: Projectile
+    damage:
+      types:
+        Piercing: 10
+
+- type: entity
+  id: Pellet410ShotgunSpread
+  noSpawn: true
+  parent: Pellet410Shotgun
+  components:
+  - type: ProjectileSpread
+    proto: Pellet410Shotgun
```

Comment:
`Pellet410Shotgun` is 1 to 1 copy of regular shotgun pellet, just use the regular pellets, not need for bloat

Reply:
Yes,  look up the 410 shotgun shell compare to a 12gauge (the one used In game)

Reply:
It's not about shells, it's about pellets. You don't need to add new pellet, especially if there is no difference between it and the one present in the game.

Reply:
Look up the 410 shotgun shell verse the 12g shell